maquetado/scss/_titles.scss

```diff
@@ -0,0 +1,19 @@
+.title {
+  align-items: center;
+  color: $black;
+  display: flex;
+  font-size: 24px;
+  height: $title-decoration-height;
+  position: relative;
+  width: 100%;
+}
+
+.title:before {
+  background: transparent url('assets/triangle.svg') no-repeat 0 0;
```

Comment:
No need for the `0 0` since the default values are OK 😄

Reply:
Done.